src/driver/sqlserver/SqlServerDriver.ts

```diff
@@ -585,7 +585,8 @@ export class SqlServerDriver implements Driver {
         return Object.keys(insertResult).reduce((map, key) => {
             const column = metadata.findColumnWithDatabaseName(key);
             if (column) {
-                OrmUtils.mergeDeep(map, column.createValueMap(this.prepareHydratedValue(insertResult[key], column)));
+                OrmUtils.mergeDeep(map, column.createValueMap(insertResult[key]));
```

Comment:
can you please explain this change?

Reply:
Changes on drivers were needed to avoid a "double hydration" which was giving issues to columns that couldn't handle it (like custom transforms). All the other drivers didn't had that .prepareHydratedValue and that's how I found why it wasn't working for those 2 drivers.